### PR TITLE
Math magicians: events

### PR DESCRIPTION
**Pull request by fbclh**
**Project requirements**

- Copy the files calculate.js and operate.js into a logic/ directory in your project.
- Analyze the files calculate.js and operate.js, be sure you understand what they do (note: you need to add [big.js] as dependency).
- Import the files in your Calculator component.
- Implement the event handlers you need to use the math logic from calculate.js and operate.js in your React component.
- Practice what you have learned until now: use state and props, pass props to child components, use lifecycle methods, lift state up.
- Test your app in the browser. It should be able to perform math operations.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,6 +6,8 @@
     "@testing-library/jest-dom": "^5.16.4",
     "@testing-library/react": "^13.1.1",
```
Comment (SajjadAhmad14) [inline package.json:7]: Please add `project description` and a set of `instructions` to run the project locally.

File: src/components/Calculator.js [hunk 2]
```diff
@@ -1,53 +1,226 @@
-import { Component } from 'react'
```
Comment (SajjadAhmad14) [inline src/components/Calculator.js:1]: Please fix all the errors which stop the app to run.
linter

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Fabio, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
Git flow is followed. :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### Please remember to tag me in your question so I can receive the notification.
**_Note: Please do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by elmar8287 — APPROVED** [review]
Hi, 

# APPROVED  👍 🥇 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**